ci/autoscaler/pipeline.yml

```diff
@@ -357,31 +377,88 @@ jobs:
   - task: deploy-autoscaler
     file: ci/ci/autoscaler/tasks/deploy-autoscaler.yml
     params:
-      <<: *acceptance-log-cache-params
+      <<: *acceptance-log-cache-syslog-params
       <<: *app-autoscaler-ops-files-log-cache-syslog
     timeout: 30m
   - task: register-broker
     file: ci/ci/autoscaler/tasks/register-broker.yml
     params:
-      <<: *acceptance-log-cache-params
+      <<: *acceptance-log-cache-syslog-params
+    timeout: 5m
+  - in_parallel:
+    - task: autoscaler-acceptance-api
+      file: ci/ci/autoscaler/tasks/run-acceptance-tests.yml
+      params:
+        <<: *acceptance-log-cache-syslog-params
+        SUITES: api
+      timeout: 15m
+    - task: autoscaler-acceptance-app
+      file: ci/ci/autoscaler/tasks/run-acceptance-tests.yml
+      params:
+        <<: *acceptance-log-cache-syslog-params
+        SUITES: app
+      timeout: 45m
+    - task: autoscaler-acceptance-broker
+      file: ci/ci/autoscaler/tasks/run-acceptance-tests.yml
+      params:
+        <<: *acceptance-log-cache-syslog-params
+        SUITES: broker
+      timeout: 15m
+
+- name: acceptance-log-cache-syslog-cf
+  public: true
+  build_logs_to_retain: 100
+  serial: true
+  on_success:
+    task: cleanup
+    file: ci/ci/autoscaler/tasks/cleanup-autoscaler.yml
+    params: &acceptance-log-cache-syslog-cf-params
+      DEPLOYMENT_NAME: ((acceptance_deployment_name_logcache_syslog_cf))
+  plan:
+  - in_parallel:
+    - get: bbl-state
+    - get: app-autoscaler-release
+      # passed: [unit-tests, integration-tests]
```

Comment:
is it intended - not to run unit-test and integration-tests before the `acceptance-log-cache-syslog-cf`